renovate.json5

```diff
@@ -0,0 +1,111 @@
+/*
+  %CopyrightBegin%
+
+  SPDX-License-Identifier: Apache-2.0
+
+  Copyright Ericsson AB 2025. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  %CopyrightEnd%
+*/
```

Comment:
This file may need to be added to the documentation mentioned before.
Simply because we hard-code vendor names, which means that the addition of any new vendor library must update this file as well. Without this being written down, we will surely forget (we will possibly forget even if written down, but at least there are steps on how to proceed)

Reply:
We should also mention that this runs as a Github App, instead of CI action.

Reply:
done